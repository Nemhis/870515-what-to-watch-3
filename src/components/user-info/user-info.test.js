import React from 'react';
import renderer from 'react-test-renderer';

import UserInfo from './user-info.jsx';

const user = {
  avatarUrl: `test/url.png`
};

describe(`Render correctly <UserInfo>`, () => {
  it(`<UserInfo> correctly render with no user`, () => {
    const tree = renderer
      .create(<UserInfo user={null}/>)
      .toJSON(`test`);

    expect(tree).toMatchSnapshot();
  });

  it(`<UserInfo> correctly render with user`, () => {
    const tree = renderer
      .create(<UserInfo user={user}/>)
      .toJSON(`test`);

    expect(tree).toMatchSnapshot();
  });
});
