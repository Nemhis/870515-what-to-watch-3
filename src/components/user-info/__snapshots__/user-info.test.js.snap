// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly <UserInfo> <UserInfo> correctly render with no user 1`] = `
<div
  className="user-block"
>
  <a
    className="user-block__link"
    href="/sign-in-page"
  >
    Sign in
  </a>
</div>
`;

exports[`Render correctly <UserInfo> <UserInfo> correctly render with user 1`] = `
<div
  className="user-block"
>
  <div
    className="user-block__avatar"
  >
    <img
      alt="User avatar"
      height="63"
      src="test/url.png"
      width="63"
    />
  </div>
</div>
`;
